stats/tinymembench

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+
+LKP_SRC = ENV['LKP_SRC'] || File.dirname(File.dirname(File.realpath($PROGRAM_NAME)))
+
+require "#{LKP_SRC}/lib/log"
+
+stats = Hash.new { |h, k| h[k] = [] }
+
+while (line = $stdin.gets)
+  case line
+  when /(standard mem|MOVSB|MOVSD|SSE2)/
+    line = line.split(':')
+    key = line[0]
+    key.lstrip!
+    key.rstrip!
+    key = key.gsub(/[)(]/, '')
+    key = key.gsub(/[ -]/, '_')
+    key = key.gsub(/$/, '_MBpS')
+    values = line[1].split(' MB/s')
+    values[0].lstrip!
+    stats[key] = values[0]
+    stats["#{key}_stddev"] = values[1].gsub(/[()% ]/, '') unless values[1].nil?
+  when /(1024|4096|16384|65536|262144|1048576)/
+    line = line.split(':')
+    line[0].lstrip!
+    line[0].rstrip!
+    values = line[1].split('/')
+    if !stats.key?("single_random_read_#{line[0]}_block_nohugepage")
```

Comment:
prefer to define a var like below to be used
`single_key = "single_random_read_#{line[0]}_block_nohugepage"`

Reply:
Used following vars and appended ```hugepage``` and ```nohugepage``` accordingly
```
single_key = "single_random_read_#{line[0]}_block"
dual_key = "dual_random_read_#{line[0]}_block"
```